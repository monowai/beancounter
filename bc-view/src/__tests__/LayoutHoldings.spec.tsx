import React from "react";
import { cleanup, render, waitForElement } from "@testing-library/react";
import "@testing-library/jest-dom/extend-expect";
import ViewHoldings from "../holdings";
import nock from "nock";

afterEach(cleanup);
const container = render(<ViewHoldings />);

nock("http://localhost:9500")
  .get("/api/test")
  .replyWithFile(200, __dirname + "/contracts/holdings.json", {
    "Access-Control-Allow-Origin": "*",
    "Content-type": "application/json"
  });

describe("<ViewHoldings />", () => {
  it("should match snapshot", async () => {
    await waitForElement(() => container.getByText("USD"));
    expect(container.debug).toMatchSnapshot();
  });
});
