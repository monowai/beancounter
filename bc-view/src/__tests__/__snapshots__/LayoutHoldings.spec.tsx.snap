// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ViewHoldings /> should match snapshot 1`] = `[Function]`;
